pages/task/read/index.js

```diff
@@ -66,11 +79,13 @@ module.exports = settings => {
         }, {})
       };
 
+      req.form.schema.decision.options = merge(req.form.schema.decision.options, res.locals.static.content.fields.options);
```

Comment:
This seems odd, you are mutating options and assigning the value to options, I would do one or the other

Reply:
also it feels a little icky that we are directly mutating the schema here - we should find a better way, ideally lookup content in the component